test/katex-spec.js

```diff
@@ -3523,10 +3523,10 @@ describe("The maxSize setting", function() {
 
 describe("The maxExpand setting", () => {
     it("should prevent expansion", () => {
-        expect`\gdef\foo{1}\foo`.toParse();
-        expect`\gdef\foo{1}\foo`.toParse(new Settings({maxExpand: 2}));
-        expect`\gdef\foo{1}\foo`.not.toParse(new Settings({maxExpand: 1}));
```

Comment:
It looks like the behavior was wrong before this PR.  It it safe to say that the following will pass now?
```
expect`\gdef\foo{1}\foo`.toParse(new Settings({maxExpand: 1}));
```

Reply:
@kevinbarabash That seems better!